migrator/migrations/2_add_constraint_on_users.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE public.users ADD CONSTRAINT users_un_unigue UNIQUE (user_name);
+ALTER TABLE public.users ADD CONSTRAINT users_email_unigue UNIQUE (email);
+ALTER TABLE public.users ADD CONSTRAINT users_phone_unigue UNIQUE (phone);
```

Comment:
As I remember you can add UNIQUE when create the table. So you don't really need these constraints to be in separate migrations.

Reply:
Fixed